src/C4Config.cpp

```diff
@@ -75,7 +76,7 @@ void C4ConfigGeneral::CompileFunc(StdCompiler *pComp)
 #ifdef _WIN32
 	pComp->Value(mkNamingAdapt(s(UserPath), "UserPath", "%APPDATA%\\LegacyClonk",                        false, true));
 #elif defined(__linux__)
-	pComp->Value(mkNamingAdapt(s(UserPath), "UserPath", "$HOME/.legacyclonk",                            false, true));
+	pComp->Value(mkNamingAdapt(s(UserPath), "UserPath", "$XDG_CONFIG_HOME/.legacyclonk",                            false, true));
```

Comment:
The indentation of the other arguments is messed up here. Whether that indentation's needed might be open up for debate, but we shouldn't break it at only one place.